### PR TITLE
issue #4012 remove everything related to subscriptions

### LINKED ISSUE #4012: remove everything related to subscriptions
like this and everything else

```ts
  private renderSubscriptionStatusHeader = (subscription: SubscriptionInfo) => {
    const isActive = subscription.level && !subscription.expired;
    const activeOrNotStr = isActive ? 'active' : 'inactive';
    $('#status-row #status_subscription').text(`s:${activeOrNotStr}`);
    if (isActive) {
      $('.logo-row .subscription .level').text('pro').css('display', 'inline-block');
    } else {
      $('.logo-row .subscription .level').text('free forever').css('display', 'inline-block');
      if (subscription.level && subscription.expired) {
        $('.logo-row .subscription .expire').text('expired').css('display', 'inline-block');
      }
    }
  }
```

### PR DESCRIPTION
**Pull request by IvanPizhenko**
This PR removes stuff related to subscriptions.

close #4012

----------------------------------

**Tests** _(delete all except exactly one)_:
- Not worth testing

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by IvanPizhenko** [comment]
@rrrooommmaaa @tomholub Ready for review

File: package.json [hunk 1]
```diff
@@ -93,4 +93,4 @@
     "url": "https://github.com/FlowCrypt/flowcrypt-browser/issues"
   },
   "homepage": "https://flowcrypt.com"
-}
\ No newline at end of file
+}
```
Comment (rrrooommmaaa) [inline package.json:96]: please don't commit unrelated changes

**Review by rrrooommmaaa — APPROVED** [review]
